include/CppUTest/Utest.h

```diff
@@ -61,7 +61,12 @@ class Utest
 
 class TestTerminator
 {
+protected:
+    static bool crashOnFail_;
+    virtual void checkCrashOnFail() const;
```

Comment:
This does not feel right. This seems to be a different TestTerminator rather than something that should be in this interface.